src/main/java/me/kavin/piped/server/handlers/auth/UserHandlers.java

```diff
@@ -27,6 +48,7 @@
 public class UserHandlers {
     private static final Argon2PasswordEncoder argon2PasswordEncoder = Argon2PasswordEncoder.defaultsForSpringSecurity_v5_8();
     private static final BCryptPasswordEncoder bcryptPasswordEncoder = new BCryptPasswordEncoder();
+    public static final Map<String, OidcData> PENDING_OIDC = new HashMap<>();
```

Comment:
This shouldn't be done, ideally, we should store these in the database, since the user could potentially make requests to two different backend instances due to load balancing.

Reply:
I have moved this to the database. There still needs to be some task to clean up the login attempts, i have included a timestamp for this purpose. I am not sure how / when the timers specified in Main work, i would appreciate some help there :)